ape_safe/config.py

```diff
@@ -1,8 +1,16 @@
 from typing import Optional
 
 from ape.api import PluginConfig
+from pydantic_settings import SettingsConfigDict
+
+from .types import SafeCacheData
 
 
 class SafeConfig(PluginConfig):
     default_safe: Optional[str] = None
     """Alias of the default safe."""
+
+    require: dict[str, SafeCacheData] = {}
```

Comment:
I am not completely sold on this name `require` but OK with whatever.
I think I would have called it `safes`, although it looks a bit funny to configure `safe:\n\tsafes:`, that is still what I would have picked

Reply:
Yeah, or `APE_SAFE_SAFES='{"alias":{...}}'

Reply:
I'm gonna skip this suggestion, I think it works